src/transformers/models/esm/convert_esm.py

```diff
@@ -65,6 +66,42 @@
     "esmfold_v1": esmfold_v1,
 }
 
+restypes = [
+    "A",
+    "R",
+    "N",
+    "D",
+    "C",
+    "Q",
+    "E",
+    "G",
+    "H",
+    "I",
+    "L",
+    "K",
+    "M",
+    "F",
+    "P",
+    "S",
+    "T",
+    "W",
+    "Y",
+    "V",
+]
```

Comment:
Should be on one line (we cna deactivate formatting if necessay for this one line).

Reply:
Fixed!